microsoft-edge/webview2/how-to/print.md

```diff
@@ -0,0 +1,744 @@
+---
+title: Printing from WebView2 apps
+description: How to print a webpage to a printer, PDF file, or PDF stream in a WebView2 app.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: webview
+ms.date: 01/12/2023
+---
+# Printing from WebView2 apps
+
+There are several different ways to print a web page in WebView2, which give you various levels of ease of implementation and customizing.
+
+| Method | Description |
+|---|---|
+| `ShowPrintUI` | Opens the WebView2 **Print Preview** dialog or the operating system's **Print** dialog. Easy to implement, minimal support for customizing. |
+| `Print` | Silently prints the current top-level document in the WebView2 using optional programmatically specified print settings to a printer. You can to build your own print preview dialog or print experience. |
+| `PrintToPdf` | Silently prints the current top-level document in the WebView2 to a PDF file. You can build your own code to print the PDF file. |
+| `PrintToPdfStream` | Silently prints the current top-level document in the WebView2 to a PDF stream. You can build your own code to print the PDF. |
```

Comment:
For both PrintToPdf and PrintToPdfStream would you use the same process to "build your own code to print the PDF"?

Reply:
Yes

Reply:
App uses same logic to print from the available pdf file/stream.